graylog2-server/src/main/java/org/graylog2/rest/resources/system/lookup/LookupTableResource.java

```diff
@@ -284,10 +290,12 @@ public LookupTableApi createTable(@ApiParam LookupTableApi lookupTable) {
     }
 
     @PUT
-    @Path("tables")
+    @Path("tables/{idOrName}")
```

Comment:
@kroepke I added this for consistency with other resources in our system. We are using this everywhere, basically. When addressing a single resource, we put the ID in the URL. I think it's a good practice and common in many other HTTP APIs.